scripts/release.sh

```diff
@@ -294,7 +294,7 @@ log "Release tags for ${new_version} created successfully and pushed to ${remote
 
 log
 # Write to a tmp file for ease of debugging.
-release_json_file=$(mktemp -t coder-release.json)
+release_json_file=$(mktemp -t coder-release.json.XXXXXX)
```

Comment:
This didn't work in dogfood without xx, on macOS this will produce literal XXXXXX but it's a temp file so who cares. 😄

Reply:
```bash
$ mktemp -t coder-release.json.XXXXXX
/var/folders/8h/5xb_vxyd23n8twprp58ds9l00000gn/T/coder-release.json.XGEmmJ
```

Seems fine on Sonoma

Reply:
You wouldn't happen to have overridden coreutils (`which mktemp`)?

```
~
❯ mktemp -t coder-release.json.XXXXXX
/var/folders/xb/7t2r1stn5lj9z5mxstz6vymc0000gn/T/coder-release.json.XXXXXX.h0piIo5nqF

~
❯ which mktemp
/usr/bin/mktemp
```

Reply:
```bash
$ which mktemp
/opt/homebrew/opt/coreutils/libexec/gnubin/mktemp
```
🤦

Reply:
That'd do it 😄